VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-network-health/participant-network-health.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit, OnDestroy, Input } from '@angular/core';
+import { EventsService } from 'src/app/services/events.service';
+import { Subscription } from 'rxjs';
+import { ParticipantHeartbeat, HeartbeatHealth } from 'src/app/services/models/participant-heartbeat';
+import { ParticipantResponse } from 'src/app/services/clients/api-client';
+import { ModalService } from 'src/app/services/modal.service';
+
+@Component({
+    selector: 'app-participant-network-health',
+    templateUrl: './participant-network-health.component.html',
+    styleUrls: ['./participant-network-health.component.scss']
+})
+export class ParticipantNetworkHealthComponent implements OnInit, OnDestroy {
+    readonly GUIDANCE_MODAL = 'more-info-modal';
```

Comment:
is it better to use const here (readonly  use for properties)

Reply:
I'll follow the same at consultation service here.